ecl/hql/hqlexpr.cpp

```diff
@@ -15061,6 +15062,10 @@ extern HQL_API bool hasUnknownTransform(IHqlExpression * expr)
         if (expr->hasAttribute(mergeTransformAtom))
             return true;
         break;
+    case no_call:
```

Comment:
I needed to add this to prevent a core... but it's possible that other changes have removed that requirement